app/src/main/java/com/example/notes2/FloatingWidgetService.kt

```diff
@@ -196,22 +200,21 @@ class FloatingWidgetService : Service() {
                 header = if (plot.length > 15) plot.substring(0, 15) + "..." else plot
             }
             val note = Note(header, plot)
-            MainActivity.notes.add(note)
```

Comment:
Are you sure you need static `notes ` list in activity? Can you create some class, that will contain this list? kinda Storage, to remove direct access with activity?